src/Components/ComparePublicationData/ComparePublicationDataModal.tsx

```diff
@@ -235,6 +244,25 @@ const ComparePublicationDataModal: FC<ComparePublicationDataModalProps> = ({
     }
   }
 
+  const successSnackBarActions = (key: any, resultId: string) => (
+    <>
+      <StyledSnackBarButton
+        startIcon={<LaunchIcon />}
+        rel="noopener noreferrer"
+        target="_blank"
```

Comment:
Usikker på om jeg skulle skrive test på dette. Har egentlig mest lyst til å gå bort fra snackbars. Økte tida snackbaren vises til 10 sekunder da.
Kunne virkelig hatt behov for en Stine på denne greia.

Reply:
jeg har fjernet snackbar på de stedene jeg har kunnet. Det gjensto den ved lagring da det krevde en del omskriving

Reply:
så gjerne fjern snackbaren ja - og vis heller en material-succes-alert

Reply:
Vanskelig å få til material success alert i denne komponenten. Modalen er lukket når snackbaren vises frem.

Reply:
jepp. det var derfor jeg ikke gjorde det selv. kan være en grei sak å legge i redux